docs/howto/partitions.rst

```diff
@@ -45,45 +45,57 @@ Required application changes
 
 To add partition support to an application, two basic changes are needed:
 
-#. Enable the feature
+#. Enable the feature.
 
-   Use the :class:`Features <craft_parts.Features>` class to specify that the
-   application will use partitions:
+   In your Application subclass, override the following method and invoke the
+   :class:`Features <craft_parts.Features>` class:
 
    .. code-block:: python
 
      from craft_parts import Features
 
-     Features.reset()
-     Features(enable_partitions=True)
+     class SnackcraftApplication(Application):
```

Comment:
While snarky and fun, it might read more straightforward for the casual reader to make this more explicit

```suggestion
     class ExampleApplication(Application):
```